pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/ActivityInstanceSql.sql.stg

```diff
@@ -66,6 +67,8 @@ select ai.activity_instance_id   as a_instance_id,
   left join numeric_answer as na on na.answer_id = ans.answer_id
   left join picklist_option__answer as pa on pa.answer_id = ans.answer_id
   left join picklist_option as po on po.picklist_option_id = pa.picklist_option_id
+  left join picklist_nested_option as npo on npo.nested_option_id = pa.picklist_option_id
```

Comment:
Since we're only pulling one column from the parent picklist option, have you considered doing a subquery instead of left joins? Since some answer rows are picklists while other answer rows are not picklist, I'm wondering what the performance is for subquery versus left-join. Same question for `AnswerDao.sql.stg`.

Reply:
I believe optimizer handles joins efficiently. Will check it out